pages/services/cassandra/2.5.0-3.11.3/operations/index.md

```diff
@@ -229,8 +229,8 @@ Restoring cluster data is similar to backing it up. The `restore-s3` plan assume
 
 Optional parameters:
 - `S3_ENDPOINT_URL`: URL of S3 compatible storage. If you want to restore from S3 compatible storage, please provide this parameter in `restore-s3` plan.
-- `AWS_SESSION_ID`: It may also be necessary to set the `AWS_SESSION_ID` depending on how you authenticate with AWS.
-- `AWS_SESSION_TOKEN`: It may also be necessary to set the `AWS_SESSION_TOKEN` depending on how you authenticate with AWS. 
```

Comment:
why remove the code format? IMO it looked better before

Reply:
well over site it was not looking good. 
https://docs.mesosphere.com/services/cassandra/2.6.0-3.11.4/operations/#backing-up-to-s3
so I changed it.
but if you want me to revert it back i will do.

Reply:
I see what you're getting at. Looks confusing to use the actual value twice. How about this as a compromise? 
```
- `AWS_SESSION_ID`: It may also be necessary to set the session ID depending on how you authenticate with AWS.
- `AWS_SESSION_TOKEN`: It may also be necessary to set the session token depending on how you authenticate with AWS.
```

Reply:
:+1: yea that will work, thanks @samvantran